ext/storage_sources/dir_store/dir_store.c

```diff
@@ -570,11 +573,12 @@ dir_store_customize_file_system(WT_STORAGE_SOURCE *storage_source, WT_SESSION *s
         cachedir.len = strlen(buf);
     }
     if ((ret = dir_store_get_directory(
-           fs->home_dir, cachedir.str, (ssize_t)cachedir.len, true, &fs->cache_dir)) != 0) {
+             fs->home_dir, cachedir.str, (ssize_t)cachedir.len, true, &fs->cache_dir)) != 0) {
```

Comment:
We still have a spacing problem here (and elsewhere).  Do you have an old version of clang-format?  Or a `.clang-format` file?  We should get to the bottom of this.